parm/use_cases/model_applications/convection_allowing_models/Point2Grid_ObsOnly_obsLSR_PracticallyPerfect.conf

```
@@ -0,0 +1,164 @@
+[config]
+## Configuration-related settings such as the process list, begin and end times, etc.
+
+# List of applications to run - Ascii2nc and Point2Grid 
+PROCESS_LIST = ASCII2NC, Point2Grid, RegridDataPlane 
+
+# time looping - options are INIT, VALID, RETRO, and REALTIME
+# If set to INIT or RETRO:
+#   INIT_TIME_FMT, INIT_BEG, INIT_END, and INIT_INCREMENT must also be set
+# If set to VALID or REALTIME:
+#   VALID_TIME_FMT, VALID_BEG, VALID_END, and VALID_INCREMENT must also be set
+LOOP_BY = INIT
+
+# Format of VALID_BEG and VALID_END using % items
+# %Y = 4 digit year, %m = 2 digit month, %d = 2 digit day, etc.
+# see www.strftime.org for more information
+# %Y%m%d%H expands to YYYYMMDDHH
+INIT_TIME_FMT = %Y%m%d%H
+
+# Start time for METplus run - must match INIT_TIME_FMT
+INIT_BEG = 2020020500
+
+# End time for METplus run - must match INIT_TIME_FMT
+INIT_END = 2020020500
+
+# Increment between METplus runs (in seconds if no units are specified)
+#  Must be >= 60 seconds
+INIT_INCREMENT = 24H
+
+# List of forecast leads to process for each run time (init or valid)
+# If unset, defaults to 0 (don't loop through forecast leads
+LEAD_SEQ = 12H
+
+# Order of loops to process data - Options are times, processes
+# Not relevant if only one item is in the PROCESS_LIST
+# times = run all wrappers in the PROCESS_LIST for a single run time, then
+#   increment the run time and run all wrappers again until all times have
+#   been evaluated.
+# processes = run the first wrapper in the PROCESS_LIST for all times
+#   specified, then repeat for the next item in the PROCESS_LIST until all
+#   wrappers have been run
+LOOP_ORDER = times
+
+# Time relative to valid time (in seconds if no units are specified) to allow files to be considered
+#  valid. Set both BEGIN and END to 0 to require the exact time in the filename
+#  Not used in this example.
+ASCII2NC_FILE_WINDOW_BEGIN = 0
+ASCII2NC_FILE_WINDOW_END = 0
+
+# Time relative to each input file's valid time (in seconds if no units are specified) for data within the file to be
+# considered valid.
+ASCII2NC_WINDOW_BEGIN = 0
+ASCII2NC_WINDOW_END = 0
+
+# Value to pass with the -format argument to ascii2nc. See MET User's Guide for more information
+ASCII2NC_INPUT_FORMAT = python
+ASCII2NC_TIME_SUMMARY_FLAG = False
+ASCII2NC_TIME_SUMMARY_RAW_DATA = False
+ASCII2NC_TIME_SUMMARY_BEG = 000000
+ASCII2NC_TIME_SUMMARY_END = 235959
+ASCII2NC_TIME_SUMMARY_STEP = 300
+ASCII2NC_TIME_SUMMARY_WIDTH = 600
+ASCII2NC_TIME_SUMMARY_GRIB_CODES = 11, 204, 211
+ASCII2NC_TIME_SUMMARY_VAR_NAMES =
+ASCII2NC_TIME_SUMMARY_TYPES = min, max, range, mean, stdev, median, p80
+ASCII2NC_TIME_SUMMARY_VALID_FREQ = 0
+ASCII2NC_TIME_SUMMARY_VALID_THRESH = 0.0
+
+
+
+# Verbosity of MET output - overrides LOG_VERBOSITY for Point2Grid only
+POINT_2_GRID_VERBOSITY = 1
```

Comment out verbosity in use case files
```suggestion
#476 POINT_2_GRID_VERBOSITY = 1
```